rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -0,0 +1,312 @@
+#
+# Copyright (c) 2019 Connexta, LLC
+#
+# Released under the GNU Lesser General Public License version 3; see
+# https://www.gnu.org/licenses/lgpl-3.0.html
+#
+
+openapi: 3.0.2
+
+info:
+  version: ${project.version}
+  title: Security Markings API
+  description: >
+    The Security Markings API is defined to provide services with the ability to
+    acquire and validate security markings.
+  contact:
+    name: Connexta
+  license:
+    name: GNU Lesser General Public License
+    url: 'http://www.gnu.org/licenses/lgpl.html'
+
+servers:
+  - url: ${rest.server.url}
+
+paths:
+  /system/markings:
+    get:
+      summary: Endpoint for acquiring the system high security markings.
```

Comment:
We technically don't need this now so we may want to wait before providing this resource and operation.